src/main/java/com/joyent/manta/monitor/Application.java

```diff
@@ -167,6 +167,10 @@ private static void copyContextToException(final Injector injector,
     }
 
     private static int validateJettyServerPort(final String jettyServerPortEnvVariable) {
+        if (System.getenv("ENABLE_TLS") != null
```

Comment:
Perhaps use `org.apache.commons.lang3.BooleanUtils` to parse the boolean value from String.